languages/brazilian/pack/src/main/res/xml/qwerty.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<Keyboard xmlns:android="http://schemas.android.com/apk/res/android"
+    android:keyWidth="10%p"
+    android:keyHeight="@integer/key_normal_height">
+    <!-- for testing shiftCodes 
+    <Row>
+        <Key android:codes="113" android:keyLabel="q" ask:mShiftedCodes="80" android:keyEdgeFlags="left"/>
+        <Key android:codes="119" android:keyLabel="w" ask:mShiftedCodes="86" ask:shiftedKeyLabel="V"/>
+        <Key android:codes="101" android:keyLabel="e" ask:mShiftedCodes="68" ask:shiftedKeyLabel="D"/>
+        <Key android:codes="114" android:keyLabel="r" ask:mShiftedCodes="81" ask:shiftedKeyLabel="Q"/>
+        <Key android:codes="116" android:keyLabel="t" ask:mShiftedCodes="83" ask:shiftedKeyLabel="S"/>
+        <Key android:codes="121" android:keyLabel="y" ask:mShiftedCodes="88" ask:shiftedKeyLabel="X"/>
+        <Key android:codes="117" android:keyLabel="u" ask:mShiftedCodes="84" ask:shiftedKeyLabel="T"/>
+        <Key android:codes="105" android:keyLabel="i" ask:mShiftedCodes="72" ask:shiftedKeyLabel="H"/>
+        <Key android:codes="111" android:keyLabel="o" ask:mShiftedCodes="78" ask:shiftedKeyLabel="M"/>
+        <Key android:codes="112" android:keyLabel="p" ask:mShiftedCodes="80" ask:shiftedKeyLabel="P" android:keyEdgeFlags="right"/>
+    </Row>
+    -->
+    <!-- original -->
+    <Row>
+        <Key android:codes="113" android:popupCharacters="1" android:keyEdgeFlags="left"/>
+        <Key android:codes="119" android:popupCharacters="2ŵω"/>
+        <Key android:codes="101" android:popupKeyboard="@xml/popup_qwerty_e"/>
+        <Key android:codes="114" android:popupCharacters="4řŕρ"/>
+        <Key android:codes="116" android:popupCharacters="5țť\u0163τ"/>
+        <Key android:codes="121" android:popupCharacters="6ýÿψ"/>
+        <Key android:codes="117" android:popupKeyboard="@xml/popup_qwerty_u"/>
```

Comment:
You have not included this xml resources. It does not compile. Also, make sure it has a `brazilian_` prefix